blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,118 @@
+.. _administration_hba:
+
+=========================
+Host Based Authentication
+=========================
+
+.. NOTE::
+
+   Host based Authentication is an Enterprise Edition feature.
+   This feature only concerns connections over :ref:`postgres_wire_protocol`.
+
+This topic explains how to configure client connections and authentication to CrateDB.
+
+By default the optional setting ``auth.host_based`` is absent and therefore host based
+authentication is disabled. The CrateDB cluster allows any unauthenticated connections.
+
+To allow authenticated access to CrateDB from specific hosts, you need to configure
+the ``auth.host_based`` setting in the ``crate.yml`` file.
+
+Allowing Authentication to CrateDB
+==================================
+
+Client access and authentication over the :ref:`postgres_wire_protocol` is
+administered by ``auth.host_based`` configuration in the ``crate.yml`` file.
+
+The general format of the ``auth.host_based`` setting is a list of remote client
+access entries, one per line,  which permit authentication to CrateDB. Each entry
+may contain no, one, or multiple fields. Allowed fields are ``user``, ``ip`` or
+``cidr``, and ``method``. The description of these fields can be found
+in :ref:`host_based_auth`.
+
+When a client sends an authentication request via the :ref:`postgres_wire_protocol`
```

Comment:
When a client connects via ... CrateDB matches the provided username and remote IP address ...